packages/bson-bench/test/utils.ts

```diff
@@ -0,0 +1,18 @@
+import * as fs from "fs/promises";
+import * as path from "path";
+
+export { exists } from "../src/utils";
+
+/**
+ * Remove all installed bson and bson-ext versions that have been installed by tests
+ */
+export async function clearTestedDeps() {
+  for await (const dirent of await fs.opendir("node_modules")) {
+    if (/^(bson-ext|bson)-(git|local)?.*$/.test(dirent.name)) {
+      fs.rm(path.join("node_modules", dirent.name), {
+        recursive: true,
+        force: true,
+      });
+    }
+  }
+}
```

Comment:
With the new project setup, this doesn't actually delete the installed modules since they get installed to the root `node_modules` folder instead of `packages/bson-bench/node_modules`